packages/common/client-utils/package.json

```diff
@@ -11,13 +11,27 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.cjs",
-	"module": "lib/index.mjs",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/indexNode.d.mts",
+				"default": "./lib/indexNode.mjs"
+			},
+			"require": {
+				"types": "./dist/indexNode.d.ts",
+				"default": "./dist/indexNode.cjs"
+			}
+		}
+	},
+	"main": "dist/indexNode.cjs",
+	"module": "lib/indexNode.mjs",
 	"browser": {
 		"./dist/indexNode.cjs": "./dist/indexBrowser.cjs",
-		"./lib/indexNode.mjs": "./lib/indexBrowser.mjs"
+		"./dist/indexNode.d.ts": "./dist/indexBrowser.d.ts",
+		"./lib/indexNode.mjs": "./lib/indexBrowser.mjs",
+		"./lib/indexNode.d.mts": "./lib/indexBrowser.d.mts"
 	},
-	"types": "dist/index.d.ts",
+	"types": "dist/indexNode.d.ts",
```

Comment:
We technically still have an index.d.ts because I did not remove index.ts, but I only kept it so `fluid-type-test-generator` doesn't fail (it seems to be hardcoded to look for `index.ts`), so seems better to have package.json reference the "real" index files.